### DISCUSSION TITLE
ADMIN PANEL more interactive 🚀

### DISCUSSION THREAD

**Discussion by j-vitali** [Sylius/Sylius#16212] [Ideas]
Hi guys, I think the admin panel of sylius can be done better...
---
<img width="1439" alt="Screenshot 2024-05-03 alle 16 38 05" src="https://github.com/Sylius/Sylius/assets/161858414/e0c26fc7-f6e9-485a-ae77-21a7e7062232">

take a look for example at Shopify (using Polaris)
---
<img width="1290" alt="KSgMu" src="https://github.com/Sylius/Sylius/assets/161858414/33702e00-23a3-40fb-b7c4-8a981e380ab9">

Where should I ask ot contribute to create a better UI & UX ? Does Sylius have a design-system?

Thanks 🙏

**Comment by CoderMaggie** [comment]
Hi @j-vitali,
please have a look at the current version of Sylius Admin Panel, to be released in the 2.0 version 👀 
https://syliusdev-demo20.bunnyenv.com/admin/ `L: sylius P: sylius`
